webknossos/examples/calculate_segment_sizes.py

```diff
@@ -0,0 +1,33 @@
+import numpy as np
```

Comment:
Please also add a respective file in `docs/src/webknossos-py/examples/` for this example and include it in the index in `docs/mkdocs.yml`.